drivers/servo/servo.c

```diff
@@ -21,26 +23,80 @@
 #include "servo.h"
 #include "periph/pwm.h"
 
+#define ENABLE_DEBUG    (0)
+#include "debug.h"
+
+#define SEC_IN_USEC     (1000000U)
```

Comment:
I found it when I `git grep`-ed during writing, but decided that the test shouldn't be dependent on timex.h just to replace a simple literal. OK to leave this in?